draft-packages/tag/KaizenDraft/Tag/Tag.scss

```diff
@@ -68,14 +72,12 @@ $small: $spacing-md;
   height: 100%;
   align-items: center;
   padding: 0 $spacing-xs;
+  margin-inline-end: -0.6625rem;
+  margin-inline-start: -0.225rem;
   margin-right: -0.6625rem;
   margin-left: -0.225rem;
```

Comment:
If `inline` is compatible, then we probably wouldn't need these lines either?

Reply:
this is correct! I should've removed them 👍